multipart/_version.py

```diff
@@ -1 +1,3 @@
-__version__ = '0.0.5'
+import pkg_resources
```

Comment:
Should we use the newer [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html#module-importlib.metadata) instead?